src/app/chat/hooks/useConversations.ts

```
@@ -107,12 +128,18 @@ export function useConversations(userId: string | null, enabled = true) {
+        if (
+            !isEnabled ||
+            !realtimeEnabled ||
+            !userId ||
+            !CONVERSATIONS_COLLECTION
+        ) {
             return;
         }
 
         let cleanupFn: (() => void) | undefined;
         let cancelled = false;
+        let pendingTimeouts: NodeJS.Timeout[] = [];
```

ESLint

[error] 'pendingTimeouts' is never reassigned. Use 'const' instead.

(prefer-const)